config.example

```diff
@@ -8,5 +8,6 @@
     "nickserv_password": "yourpasswd",
     "auxchans": [],
     "extendedcommands": ["jaden","excuse","dong"],
-    "topmodifier": 0.05
+    "topmodifier": 0.05,
+    "repolink": "https://github.com/tonyravioli/dongerdong"
```

Comment:
Why would this be a config variable? If somebody forks this project, they can just edit the sauce.